### PR TITLE
New package: kitty-0.9.0

### PR DESCRIPTION
**Pull request by julio641742**

### CODE REVIEW

**Comment by SolitudeSF** [comment]
[0.9.0 just released](https://github.com/kovidgoyal/kitty/releases/tag/v0.9.0)

**Comment by SolitudeSF** [comment]
does it perform normally with shebangs pointing to python2?

**Comment by julio641742** [comment]
You are right @SolitudeSF about the shebangs, it is fixed in the latest commit plus the version update

**Comment by julio641742** [comment]
Package ready to be merged! :+1:

**Comment by SolitudeSF** [comment]
you should probably squash commits into one

File: srcpkgs/kitty/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+do_build() {
+	if [ -n "$CROSS_BUILD" ]; then
+		export CFLAGS="-I${XBPS_CROSS_BASE}/usr/include/python3.6m"
+		sed -i "s|-fpie|-fpie ${CFLAGS}|g" setup.py
+	fi
+	case "$XBPS_TARGET_MACHINE" in
+		*-musl)
+			sed -i 's/pow10/pow_10/g' kitty/parser.c;;
+	esac
+	python3 setup.py linux-package --prefix=${DESTDIR}/usr
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: Stupid question, I'm sure someone asked.... why no build_style to take care of this stuff?

File: srcpkgs/kitty/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+pkgname=kitty
+version=0.9.0
+revision=1
+python_version=3
+hostmakedepends="ncurses pkg-config python3 python3-devel wayland-devel wayland-protocols"
+makedepends="gettext-devel glfw-devel harfbuzz-devel libXcursor-devel libXinerama-devel
+libxkbcommon-devel libXrandr-devel python3-devel wayland-devel wayland-protocols"
+depends="kitty-terminfo"
+conf_files="/etc/kitty/kitty.conf"
+short_desc="A modern, hackable, featureful, OpenGL based terminal emulator"
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: `s/A modern/Modern/`

**Comment by Piraty** [comment]
Maybe you want to take some advice from [previous attempts to package kitty](https://github.com/voidlinux/void-packages/pulls?q=kitty), especially the one by @laserbat.
Although i find it interesting why so many PRs on the same package got closed by their contributors and never merged...

**Comment by SolitudeSF** [comment]
@Piraty 2 of them stopped using void, and 3 - kitty.

**Comment by julio641742** [comment]
I squashed all commits into one as suggested by @SolitudeSF and changed the desc @Vaelatern

File: srcpkgs/kitty/template [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'kitty'
+pkgname=kitty
+version=0.9.0
+revision=1
+python_version=3
```
Comment (cr6git) [inline srcpkgs/kitty/template]: why this variable `python_version=3` ??

Comment (julio641742) [reply srcpkgs/kitty/template]: If `python_version=3` is not set, the script `03-rewrite-python-shebang` would convert the shebang to `python2` which is not correct.
```
kitty-0.9.0_1: running pre-pkg hook: 03-rewrite-python-shebang ...
    Shebang converted to '#!/usr/bin/python2': /usr/lib/kitty/__main__.py
...
```

**Comment by julio641742** [comment]
Thanks @cr6git for the feedback, it is now ready for another review :+1:

**Comment by julio641742** [comment]
Is there anything else I need to fix before this gets merged?

**Comment by cr6git** [comment]
@julio641742 sorry, I'm not at home currently... and yes, you must change 2 things: version 0.9.1 was released a couple of hours ago, and the license is `GPL-3.0-or-later`.

**Pull request closed without merging**